packages/compatibility/package.json

```diff
@@ -21,24 +21,24 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "dependencies": {
-    "@apollo/rover": "^0.10.0",
+    "@apollo/rover": "^0.11.1",
     "debug": "^4.3.4",
     "execa": "^5.1.1",
```

Comment:
TODO: starting with v6 `execa` is pure ESM module and requires some additional code changes, will tackle it in a separate PR